app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,16 @@
+# Output devise error message in format
+module DeviseHelper
+  def devise_error_messages!
```

Comment:
It is devise defined method name..here is to override the method and format the error message